tests/python_frontend/binop_replacements_test.py

```diff
@@ -397,6 +397,7 @@ def mixed(A: dace.int64[M, N], B:dace.int64):
     return 5j + dace.complex64(M) + A[0, 0] + 32 + A[0, 1] + B + 2 + M + N
 
 
+@pytest.mark.skip
```

Comment:
This test should be passing. What seems to be the issue?

Reply:
I actually don't remember. We can try it in a separate PR